test/models/VatObligationsSpec.scala

```diff
@@ -21,56 +21,208 @@ import play.api.libs.json.{JsValue, Json}
 
 class VatObligationsSpec extends SpecBase {
 
-  val vatObligationsModel: VatObligations =
-    VatObligations(
-      Seq(VatObligation(
-        ObligationIdentification("555555555", "VRN"),
-        Seq(
-          ObligationDetail("F", "1980-02-03", "1980-04-05", Some("1980-02-02"), "1980-04-08", "17AA"),
-          ObligationDetail("F", "1981-02-03", "1981-04-05", Some("1981-02-02"), "1981-04-08", "18AA")
+
+  "VatObligations when there is a single obligation" should {
+
+    val singleObligationInputJson: JsValue = Json.parse(
```

Comment:
This test says "when there is a single obligation" but there are multiple obligations in this JSON.

Reply:
No it's correct.
There is a single obligation. 
There are multiple obligation details.
This differentiates it from the case where there are multiple obligations AND multiple obligation details